src/components/create-contingency-list-form.js

```diff
@@ -53,8 +53,6 @@ export const CreateContingencyListForm = ({ open, onClose }) => {
     const [contingencyInvalid, setContingencyInvalid] = useState(false);
```

Comment:
maybe rename `contingencyInvalid` to `contingencyValid`. Must be coherent everywhere for a same state meaning because we have : 
* `contingencyInvalid`
* `filterInvalid`
but
* `studyValid`